package.json

```diff
@@ -25,6 +25,8 @@
     "fix-lint": "prettier --write \"{,!(node_modules)/**/}*.{js,jsx}\"",
     "eslint": "eslint \"./src/**/*.{js,jsx}\"",
     "eslint-fix": "eslint \"./src/**/*.{js,jsx}\" --fix",
+    "stylelint": "stylelint \"**/*.scss\"",
+    "stylelint-fix": "stylelint \"**/*.scss\" --fix",
```

Comment:
Not sure this one is needed (I know we have an `eslint-fix`, but I don't think that one's needed as well).